concrete/src/Page/Page.php

```diff
@@ -3333,6 +3333,20 @@ public function updateDisplayOrder($do, $cID = 0)
         }
         $db = Database::connection();
         $db->executeQuery('update Pages set cDisplayOrder = ? where cID = ?', [$do, $cID]);
+
+        // Because the display order of another page can be changed,
+        // the page object is retrieved first in order to pass it to the event.
+        $page = $this;
+        if ($cID !== $this->getCollectionID()) {
+            $page = static::getByID($cID);
```

Comment:
This is relying on local cache, instead it''d be better to just query for the value before updating it and only triggering the event if it changes.

Reply:
> instead it'd be better to just query for the value

What do you mean with 'the value'?

Reply:
I mean just run a `select cDisplayOrder from Pages where cID=?` before changing it in the database

Reply:
Yes, but I intentially didn't add that to this PR, because it's another problem. I'd be happy to add that to another PR though.